charts/velero/Chart.yaml

```diff
@@ -1,9 +1,9 @@
 apiVersion: v2
-appVersion: 1.9.2
+appVersion: 1.10.0
 kubeVersion: ">=1.16.0-0"
 description: A Helm chart for velero
 name: velero
-version: 2.32.1
+version: 2.32.2
```

Comment:
Since the Velero bumps the minor version, the helm chart should bump the minor version.

Reply:
Thanks, I'll modify it

Reply:
Modified